### ISSUE TITLE
I wonder  whether there are any embedding models in Chinese?

### ISSUE THREAD

**Issue by chengzi-big** [UKPLab/sentence-transformers#2851]
I wonder  whether there are any embedding models in Chinese?

**Comment by anshuchen** [comment]
> I wonder whether there are any embedding models in Chinese?

Yes, many! You can check the MTEB leaderboard for models that perform well in Chinese. https://huggingface.co/spaces/mteb/leaderboard

**Comment by tomaarsen** [comment]
Indeed! You can also search here for Sentence Transformer models known to work well for Chinese: https://huggingface.co/models?library=sentence-transformers&language=zh&sort=trending

There's quite a few solid options.

- Tom Aarsen
